### DISCUSSION TITLE
Mute Indicator Light

### DISCUSSION THREAD

**Discussion by cloaked2222** [SaifAqqad/AHK_MicMute#45] [Ideas]
Please add a feature to control a USB light. The light should turn red when muted, and white when unmuted.

Id suggest using the blink(1) light, which works on PC and MacOS. Here's an example: https://thingm.com/blog/2021/8/27/setup-a-mute-indicator-light-for-zoom-w-hammerspoon-amp-blink1

**Comment by SaifAqqad** [comment]
Sorry but this is out of scope for me, I don't use (or have) this and therefore I cannot do it.
PRs are welcomed though.

**Comment by cloaked2222** [comment]
No worries, I finally found a hardware mute button/light here: muteme.com
